packages/atlas/src/components/_video/VideoTilePublisher/VideoTilePublisher.tsx

```diff
@@ -47,11 +47,19 @@ export const DELAYED_FADE_CLASSNAME = 'delayed-fade'
 
 export const VideoTilePublisher: FC<VideoTilePublisherProps> = memo(
   ({ id, onEditClick, onDeleteVideoClick, onReuploadVideoClick, onMintNftClick, titlesInSync }) => {
+    const [videoTitleMap, setVideoTitleMap] = useState('')
```

Comment:
What means `videoTitleMap`? In the end, It's not a map, it's a string 😅 . I'm little confused reading this code, and what this effect below is doing. Could you explain?

Reply:
`...Map` because this is mapped value of video title, we can change that naming np. 

The problem here is that from ypp backend we don't receive joystream video id, so we need to refetch videos by the title of video. This effect allows to access video title in `pollInterval`.